docs/src/openapi-v1.yaml

```
@@ -2338,12 +2338,15 @@ components:
         phase:
           type: string
           enum:
-          - opening
+          - starting
           - ending
+          - delay
```

```suggestion
          - startingPeriod
          - endingPeriod
          - vrfDelay
```
I think we should maximize alignment with https://github.com/paritytech/polkadot/blob/60341e125563cbf655ab686ac5cef370ac33d16d/runtime/common/src/traits.rs#L150-L164

---

> i will not that im not sure what to do for casing on `vrfDelay`

I think `vrfDelay` is fine. It could be `VRFDelay` as well, given it's an acronym? But `vrfDelay` sounds better to me.

---

I agree we should stay close to the Rust source. I also think it's a bit unfortunate we went with `-ing` rather than `StartPeriod`/`EndPeriod`.

---

Yea I like the idea of `vrfDelay`. One thing I just want to confirm too, if we append `Period` to `starting` and `ending` it will be a breaking change for the API. Just something to note. If you both think its appropriate while its still an experimental endpoint I am down to make the change.